src/blob.rs

```diff
@@ -17,11 +17,56 @@ struct Header {
     flags: u64,
 }
 
-impl<T> Blob<T> where T: Default {
+impl<K> Blob<K> where K: Default {
+    /// # Description
+    /// Create new blob from file
+    // @TODO more useful result
     pub fn from_file<P: AsRef<Path>>(_path: P) -> Result<Self, ()> {
         // @TODO implement
         Ok(Self::default())
     }
+
+    /// # Description
+    /// Writes given slice to file, returns number of written bytes
+    // @TODO more useful result
+    pub fn write(&mut self, _key: &K, _value: &[u8]) -> Result<usize, ()>  {
```

Comment:
I don't think we are interested that exact number of bytes has been writen to the disk. may be err in return will be enough.